### DISCUSSION TITLE
How to add icon in front of tree component

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4228] [Q&A]
# How to add icon in front of tree component

```
&widget.Tree{}
```

# This effect
![截屏2023-09-10 15 14 05](https://github.com/fyne-io/fyne/assets/1819457/7afa0526-4428-4f15-8e22-93ca716c0683)

**Comment by andydotxyz** [comment]
I don't think this is specific to tree is it?
Put an icon in front of a label then use a `HBox` container with `Icon` then `Label` content.
